torchdrivesim/rendering/cv2.py

```diff
@@ -21,6 +20,7 @@ class CV2Renderer(BirdviewRenderer):
     Renderer based on OpenCV. Slow, but easy to install. Renders on CPU.
     """
     def __init__(self, cfg: CV2RendererConfig, *args, **kwargs):
+        import cv2
```

Comment:
I think you also need to import it inside the `render_rgb_mesh` function. Otherwise it's out of scope there.